src/interactions/hoverInteraction.ts

```diff
@@ -47,11 +48,22 @@ export module Interaction {
     public _anchor(component: Hoverable, hitBox: D3.Selection) {
       super._anchor(component, hitBox);
       this._dispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> (<any> this._componentToListenTo)._element.node());
+      this._dispatcher.onMouseMove("hover" + this.getID(), (p: Point) => this._handlePointerEvent(p));
 
-      this._dispatcher.onMouseMove("hover" + this.getID(), (p: Point) => this._handleMouseEvent(p));
+      this._touchDispatcher = Dispatcher.Touch.getDispatcher(<SVGElement> (<any> this._componentToListenTo)._element.node());
+      this._touchDispatcher.onTouchStart("hover" + this.getID(), (p: Point, e: TouchEvent) => this._handleTouchEvent(p, e));
+      this._touchDispatcher.onTouchMove("hover" + this.getID(), (p: Point, e: TouchEvent) => this._handleTouchEvent(p, e));
     }
 
-    private _handleMouseEvent(p: Point) {
+    private _handleTouchEvent(p: Point, e: TouchEvent) {
+      var translatedP = this._translateToComponentSpace(p);
+      if (this._isInsideComponent(translatedP)) {
+        e.preventDefault();
+      }
+      this._handlePointerEvent(p);
```

Comment:
There's some logic that gets executed if the event occurs outside the `Component`'s area -- basically a final `hoverOut` is sent.